plugins/commands/compare/src/main/kotlin/CompareCommand.kt

```diff
@@ -158,12 +169,22 @@ private class InvariantInstantSerializer : StdSerializer<Instant>(Instant::class
     }
 }
 
+private class EpochInstantDeserializer : StdDeserializer<Instant>(Instant::class.java) {
+    override fun deserialize(parser: JsonParser, context: DeserializationContext): Instant =
+        Instant.EPOCH.also { parser.codec.readTree<JsonNode>(parser) }
+}
+
 private class InvariantEnvironmentSerializer : StdSerializer<Environment>(Environment::class.java) {
     override fun serialize(value: Environment, gen: JsonGenerator, provider: SerializerProvider) {
         gen.writeString("invariant")
     }
 }
 
+private class DefaultEnvironmentDeserializer : StdDeserializer<Environment>(Environment::class.java) {
+    override fun deserialize(parser: JsonParser, context: DeserializationContext): Environment =
+        Environment().also { parser.codec.readTree<JsonNode>(parser) }
```

Comment:
See above.

Reply:
`readTree` works only for `JsonNode` ([docs](https://javadoc.io/static/com.fasterxml.jackson.core/jackson-core/2.9.9/com/fasterxml/jackson/core/ObjectCodec.html#readTree-com.fasterxml.jackson.core.JsonParser-))